portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +28,32 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Aymar!");
+    /*These lines used to return a Hello Aymar message*/
+    //response.setContentType("text/html;");
+    //response.getWriter().println("Hello Aymar!");
```

Comment:
Nit: We usually try to avoid committing commented out code

Reply:
removed comments from branch